### PR TITLE
Refactor trust view command into a --pretty flag on trust inspect

### PR DESCRIPTION
**Pull request by n4ss**
Signed-off-by: Nassim 'Nass' Eddequiouaq <eddequiouaq.nassim@gmail.com>

**- What I did**
* Remove `docker trust view` which contains the same information as `docker trust inspect` but in a pretty-print format.
* Moved the pretty-print format feature to a `--pretty` flag on `docker trust inspect` (same as `docker service inspect [--pretty]`).
* Cleaned-up the pretty-print ouput format.
* Adapted the tests of `docker trust view` to test `docker trust inspect --pretty` instead with the new output format (some duplicates tests are now present because the two original commands had some similar tests)
*  `docker trust inspect --pretty` (old `docker trust view`) now accepts several images as parameters to inspect (used to be limited to 1, but `docker trust inspect` accepts several images so we had to keep consistency)

**- How I did it**
For now (I'll refactor that later), `docker trust inspect` executes the old codepath if no `--pretty` flag is provided on the cli, otherwise executes `docker trust view`'s codepath for each image arguments.

**- How to verify it**
`$ make -f docker.Makefile test`

and

`$ docker-dev trust inspect [--pretty] nginx:latest redis:latest docker:dind`

**- Description for the changelog**
* Moved the pretty-print format feature of `docker trust view` to a `--pretty` flag on `docker trust inspect`. The `docker trust view` has been removed.

### CODE REVIEW

**Comment by n4ss** [comment]
cc @vdemeester @thaJeztah @dnephin

File: cli/command/trust/inspect.go [hunk 1]
```diff
@@ -11,24 +12,56 @@ import (
 	"github.com/theupdateframework/notary/tuf/data"
 )
 
+type inspectOptions struct {
+	remotes []string
+	/* FIXME(n4ss): this is consistent with `docker service inspect` but we should provide
```
Comment (dnephin) [inline cli/command/trust/inspect.go]: go comments are generally done with `//` even if they are multi-line. `/* */` is only used for package comments.

File: cli/command/trust/inspect_pretty_test.go [hunk 2]
```diff
@@ -17,24 +17,21 @@ import (
 	"github.com/theupdateframework/notary/tuf/data"
 )
 
+/* TODO(n4ss): remove common tests with the regular inspect command */
```
Comment (dnephin) [inline cli/command/trust/inspect_pretty_test.go]: same nit about comment

**Comment by dnephin** [comment]
some build failures as well

**Pull request merged**